pkg/infrastructure/openstack/clusterapi/clusterapi.go

```diff
@@ -116,6 +125,21 @@ func (p Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 		}
 	}
 
+	bootstrapPort, err := getPortForInstance(networkClient, *bootstrapMachine.Spec.InstanceID)
+	if err != nil {
+		return err
+	}
+
+	bootstrapFIP, err := createFIP(networkClient, "bootstrap", infraID, ospCluster.Status.Network.ID)
```

Comment:
See my comment at https://github.com/shiftstack/installer/commit/585be0f0c86844ab8e69a53fa9dbdf957b1442c0#r138625202.

We should create and assign the FIP with a single API call.